tests/cupy_tests/sorting_tests/test_search.py

```diff
@@ -238,3 +238,155 @@ class TestFlatNonzero(unittest.TestCase):
     def test_flatnonzero(self, xp, dtype):
         array = xp.array(self.array, dtype=dtype)
         return xp.flatnonzero(array)
+
+
+@testing.gpu
+class TestNanArgMin(unittest.TestCase):
+
+    @testing.for_all_dtypes(no_complex=True)
+    @testing.numpy_cupy_allclose()
+    def test_nanargmin_all(self, xp, dtype):
+        a = testing.shaped_random((2, 3), xp, dtype)
+        return a._nanargmin()
```

Comment:
`_nanargmin` attribute is private, so it does not need to be tested.
```suggestion
        return cupy.nanargmin(a)
```

Reply:
Oh yeah that's right. Changed it now. Thanks